extensions/SemanticForms/includes/SF_ParserFunctions.php

```diff
@@ -194,7 +194,7 @@ static function renderFormLink ( &$parser ) {
 		return $parser->insertStripItem( self::createFormLink( $parser, $params, 'formlink' ), $parser->mStripState );
 	}
 
-	static function renderFormRedLink ( &$parser ) {
+	static function renderFormRedLink ( $parser ) {
 		$params = func_get_args();
 		array_shift( $params ); // We don't need the parser.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)